packages/mdc-fab/_mixins.scss

```diff
@@ -96,4 +99,19 @@
   align-items: center;
   justify-content: center;
   width: 100%;
+  will-change: transform;
+  transition: mdc-animation-enter(transform, 180ms, 90ms);
+}
+
+@mixin mdc-fab--exited_ {
+  transform: scale(0);
+  transition:
+    opacity 15ms linear 150ms,
+    mdc-animation-exit-permanent(transform, 180ms);
```

Comment:
is this the same 180ms from above? If so, maybe reference a private variable directly

Reply:
I'm not sure that these are meant to be strongly correlated (one is the duration of just the FAB's icon transitioning in, while the other is the duration of the entire FAB transitioning out). I'll ping one of our motion designers to find out. For now I'd rather err on the side of not factoring these together.

Reply:
I got confirmation that these two should not be correlated to a common variable, so I'll leave my updated PR as-is, only correlating the enter durations.